### PR TITLE
XW-176 | Fix icons for tooltips inside CC tool

### PR DESCRIPTION
**Pull request by vforge**

### CODE REVIEW

File: front/styles/auth/module/_form-icons.scss [hunk 1]
```diff
@@ -10,9 +12,8 @@
 	top: 0;
 	width: 24px;
 
-	&.info {
-		// TODO: fix Gulp tasks concerning serving images from static domains
-		background-image: url('/mercury-static/images/info.svg');
+	&.help {
+		background-image: url($help-svg);
```
Comment (vforge) [inline front/styles/auth/module/_form-icons.scss]: @bkoval Would you like to do your icons in the same way - it works on devbox now :)

File: front/styles/auth/module/_form-icons.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
...help-svg: 'data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" width="26" height="26" viewBox="0 0 26 26"><g fill="none" fill-rule="evenodd"><path stroke="#1A5EB8" d="M13 1C6.373 1 1 6.373 1 13c0 6.628 5.373 12 12 12s12-5.372 12-12c0-6.627-5.373-12-12-12z"/><g fill="#1A5EB8"><path d="M12.596 16.514c-.473-.002-.9.142-1.214.437-.32.297-.48.72-.477 1.186-.004.467.156.89.475 1.19.31.3.734.448 1.204.448h.012c.493.002.932-.14 1.25-.448.318-.3.478-.723.475-1.19.004-.467-.156-.89-.477-1.185-.32-.298-.755-.438-1.247-.436M13.188 6.328c-.837 0-1.563.1-2.178.312-.597.205-1.13.466-1.596.785l-.218.15 1.15 2.204.276-.193c.138-.095.3-.19.482-.284.18-.093.367-.175.565-.246.194-.07.39-.125.583-.163.19-.037.37-.054.534-.054.544-.003.86.12.992.27.166.182.258.42.26.773 0 .266-.062.472-.177.645-.136.204-.308.405-.52.6-.224.206-.465.42-.725.646-.283.245-.533.534-.75.862-.224.34-.395.734-.518 1.184-.086.313-.126.662-.126 1.045 0 .192.01.393.03.603l.026.27h2.5v-.3c0-.454.078-.808.212-1.063.15-.29.33-.54.54-.76.222-.224.466-.438.733-.638.296-.222.568-.478.814-.768.255-.3.465-.66.632-1.075.176-.438.257-.965.257-1.583.003-.96-.346-1.778-1.03-2.36-.682-.586-1.616-.865-2.752-.864z"/></g></g></svg>';
```
Comment (bognix) [inline front/styles/auth/module/_form-icons.scss]: please add comment with link to ticket where svg issue on devbox should be fixed

**Comment by bognix** [comment]
LGTM

File: front/styles/auth/module/_form-icons.scss (hunk 2 above)
Comment (vforge) [inline front/styles/auth/module/_form-icons.scss]: Done.

File: front/styles/auth/module/_form-icons.scss (hunk 1 above)
Comment (bkoval) [inline front/styles/auth/module/_form-icons.scss]: @vforge Yeah, that makes sense to make it running on dev environments until the assets are uploaded to production servers. However, we still don't serve auth pages assets through nocookie domain (https problem) + this solution makes the same asset be kept in two places (encoded and as a separate file). As soon as we move to using nocookie on auth pages, we could start considering implementation of a tools for inlining assets into CSS (as we do in the app stack)

**Comment by tomasznapieralski** [comment]
LGTM

File: front/styles/main/component/_curated-content-editor.scss [hunk 3]
```diff
@@ -27,13 +27,12 @@ $image-photo-icon-size: 32px;
...
-.curated-content-editor-help {
+.curated-content-editor-info {
 	position: absolute;
 	right: 1em;
 	top: 1em;
 
 	svg {
-		fill: $color-gray-dark;
 		height: 32px;
 		width: 32px;
```
Comment (Warkot) [inline front/styles/main/component/_curated-content-editor.scss]: According to specs in Zeplin width and height should have just `24px` which is `1.5em`.

File: front/styles/main/component/_curated-content-editor.scss [hunk 4]
```diff
@@ -27,13 +27,12 @@ $image-photo-icon-size: 32px;
 	padding: $button-padding-vertical $button-padding-horizontal;
 }
 
-.curated-content-editor-help {
+.curated-content-editor-info {
 	position: absolute;
 	right: 1em;
 	top: 1em;
```
Comment (Warkot) [inline front/styles/main/component/_curated-content-editor.scss]: When setting `1.5em` width and height due to https://github.com/Wikia/mercury/pull/1320/files#r38653814, `top` should be increased to `1.5em`.

**Comment by Warkot** [comment]
LGTM
Tested.
Merging.

**Pull request merged**